src/gui/Settings.cc

```diff
@@ -154,10 +185,12 @@ SettingsEntryEnum Settings::octoPrintFileFormat(
     createFileFormatItems({FileFormat::ASCII_STL, FileFormat::BINARY_STL, FileFormat::_3MF, FileFormat::OFF}),
     fileformat::info(FileFormat::ASCII_STL).description);
 
-SettingsEntryString Settings::localSlicerExecutable("printing", "localSlicerExecutable", "");
-SettingsEntryEnum Settings::localSlicerFileFormat(
-    "printing", "localSlicerFileFormat", createFileFormatItems(fileformat::all3D()),
+SettingsEntryString Settings::localAppExecutable("printing", "localAppExecutable", "");
+SettingsEntryString Settings::localAppTempDir("printing", "localAppTempDir", "");
+SettingsEntryEnum Settings::localAppFileFormat(
+    "printing", "localAppFileFormat", createFileFormatItems(fileformat::all3D()),
```

Comment:
Since local apps don't inherently only deal with 3D file formats, should we rather enable all formats here? (`fileformat::all()`)

Reply:
Yes, that would make sense, even more so once we can get multiple configurable at some point.

Reply:
As discussed, I'll update in some later patch...